assets/js/ajax-core.js

```diff
@@ -129,22 +129,43 @@ jQuery(function($){
 
                 ev.preventDefault();
 
-                var form	=	$(el),
-                    buttons = 	form.find(':submit');
-
-                if( form.data('_cf_manual') ){
-                    form.find('[name="cfajax"]').remove();
+                var $form	=	$(el),
+                    buttons = 	$form.find(':submit');
+                var pending = [];
+
+				/**
+                 * This event is triggered directly before the request for form submission is made
+                 *
+                 * Runs after cf.form.submit
+                 *
+                 * @since 1.8.0
+				 */
+				$( document ).trigger( 'cf.form.request', {
+                    $form: $form,
```

Comment:
@New0 Should we be sharing phpStorm settings? Not sure mine are to blame or yours, but this is not optimal.

 We need eslint or prettier #2485

Reply:
Absolutely @Shelob9, do you to share your code style settings ?
https://www.jetbrains.com/help/idea/copying-code-style-settings.html

Reply:
LOL I have that post in my list of things to do. I'll try it out today.